theapacheway.md

```diff
@@ -0,0 +1,7 @@
+# Open Source The Apache Way
+
+The second FOSS Hackathon by EU FOSSA 2 was a good opportinity for outside observers to get a sense of how Open Source works "The Apache Way". 
+
+Many Apache Project Management Committee members from different projects were part of the event and showed attendees not part of any Apache community how projects are working at the Apache Software Foundation. Just a week after the Apache Software Foundations 20 year anniversary the Hackathon was an opportunity for some attendees to see how an Open Source community works for the first time. A few members from European institutions got the chance to talk directly to the people involved in the Open Source community which should be a good start for understanding better how to support Open Source projects in the future.
+
+The Apache Way is all about community. Decission are made together and take place in public. It can be hard to understand to understand how and why Open Source and Apache works from the outside. When you start talking to Apache contributors and work with them, even if it is not producing code, you start to get a feeling why it is successful and important.
```

Comment:
"Decisions" are ...
"to understand" (repeated)
"how and why Open Source and Apache work"
"get a feeling of why"